src/rules/root-no-standard-properties/index.js

```diff
@@ -4,6 +4,8 @@ import {
   validateOptions,
 } from "../../utils"
 
+const RE_VALID_CSS_PROPERTY = /^[a-z-]+$/
```

Comment:
If we have a standard, reused way of checking whether a property is a valid CSS property, we could put it in `utils/` with the other reused functions like that.